packages/@o3r/rules-engine/builders/rules-engine-extractor/index.ts

```diff
@@ -73,18 +73,18 @@ export default createBuilder<RulesEngineExtractorBuilderSchema>(async (options,
 
   /** Facts from the current project */
   const newFactList = (await Promise.all(
-    (await Promise.all(options.factFilePatterns.map((pattern) => pGlob(pattern, {cwd: context.currentDirectory}))))
+    (await Promise.all(options.factFilePatterns.map((pattern) => (pGlob as any).default(pattern, {cwd: context.currentDirectory}))))
```

Comment:
We should maybe find a real solution for this issue

Reply:
try `import pGlob from 'globby';` instead of `import * as pGlob from 'globby';`